packages/nextra-theme-blog/src/meta.tsx

```diff
@@ -4,12 +4,17 @@ import { useBlogContext } from './blog-context'
 import ThemeSwitch from './theme-switch'
 import { split } from './utils/get-tags'
 import { getParent } from './utils/parent'
+import { useTheme } from 'next-themes'
+
 
 export default function Meta(): ReactElement {
   const { opts, config } = useBlogContext()
   const { author, date, tag } = opts.frontMatter
   const { back } = getParent({ opts, config })
   const tags = tag ? split(tag) : []
+  const { resolvedTheme } = useTheme()
+  const textColor =
+    resolvedTheme === 'dark' ? 'nx-text-gray-400' : 'nx-text-gray-600'
```

Comment:
```suggestion
    'dark:nx-text-gray-400 nx-text-gray-600'
```
Use dark modifier instead, also refactor your previous usages